### ISSUE TITLE
[Feature Request] Prisma 5.17 support

### ISSUE THREAD

**Issue by tony19** [zenstackhq/zenstack#1598]
**Is your feature request related to a problem? Please describe.**

Can we add support for [Prisma 5.17](https://github.com/prisma/prisma/releases/tag/5.17.0)? Similar to #1536

Otherwise, I get an error when trying to install `@prisma/client@5.17.0`:

```
npm error While resolving: nextjs-template@0.1.0
npm error Found: @prisma/client@5.17.0
npm error node_modules/@prisma/client
npm error   @prisma/client@"^5.17.0" from the root project
npm error
npm error Could not resolve dependency:
npm error peer @prisma/client@"5.0.0 - 5.16.x" from @zenstackhq/runtime@2.3.2
npm error node_modules/@zenstackhq/runtime
npm error   @zenstackhq/runtime@"^2.3.2" from the root project
npm error
npm error Fix the upstream dependency conflict, or retry
npm error this command with --force or --legacy-peer-deps
npm error to accept an incorrect (and potentially broken) dependency resolution.
```

**Describe the solution you'd like**
Update dependency versions, as was done in #1541 

**Describe alternatives you've considered**
* `overrides` in `package.json`
* `npm install` with `--force` or `--legacy-peer-deps`

**Additional context**
None

**Comment by ymc9** [comment]
WIP 😄

**Comment by ymc9** [comment]
Fixed in 2.3.3
